HaxeManual/04-class-field.tex

```diff
@@ -140,7 +140,7 @@ \subsection{Rules for getter and setter}
 
 Visibility of accessor methods has no effect on the accessibility of its property. That is, if a property is \expr{public} and defined to have a getter, that getter may be defined as \expr{private} regardless.
 
-Both getter and setter may access their physical field for data storage. The compiler ensures that this kind of field access does not go through the accessor method if made from within the accessor method itself, thus avoiding infinite recursion:
+Both getter and setter may access their physical field for data storage. The compiler ensures that this kind of field access does not go through the accessor method is made from within the accessor method itself, thus avoiding infinite recursion:
```

Comment:
No, it's saying the physical access is used *when* made from within the accessor method. It was correct as it was.